### DISCUSSION TITLE
2 minute delay for premarket data (maybe in market too?)

### DISCUSSION THREAD

**Discussion by martin0** [ranaroussi/yfinance#1475] [General]
Hi 
I've noticed there's a delay in the retrieval of pre-market data (~2 minutes)

So for example , requesting 1 minute interval data, I don't see the 08:00 EST candle until 08:02 EST.

Do other people experience this?
Does it improve with YahooFinancePlus?

Thanks
Martin

**Comment by andreyversh** [comment]
Hello, Martin, please share the code sampe to get data for premarket. I use 
`df = yf.download(tickers='GOOG', period=timeslot, interval='1m', rounding=True, group_by='Ticker', prepost=True)`,
where _timeslot_ covered premarket time from 4:00 to 09:30 EST, but got all data for premarket only, when main session is started.

**Comment by martin0** [comment]
I used start time just an hour old

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
